crates/wash-cli/README.md

```diff
@@ -132,6 +132,10 @@ There are three main sections of a `wasmcloud.toml` file: common config, languag
 | cargo_path  | string | `which cargo` | The path to the cargo binary            |
 | target_path | string | ./target      | Path to cargo/rust's `target` directory |
 
+#### Language Config - [other]
+
+If you are using a language other than Rust or Go, you can designate your language as any string. Since `wash` will be unable to infer the build toolchain, you must supply the `build_command` and the path to the `build_artifact` so that the artifact can be built and found before signing.
```

Comment:
```suggestion
If you are using a language other than Rust or Go, you can designate your language as any string (ex. `javascript`). 

Since `wash` will be unable to infer the build toolchain from the language you provide, you must supply the `build_command` and the path to the `build_artifact` so that the artifact can be built and found before signing.
```